src/main/java/net/sf/jabref/logic/importer/fetcher/DiVA.java

```diff
@@ -0,0 +1,80 @@
+package net.sf.jabref.logic.importer.fetcher;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Optional;
+
+import net.sf.jabref.importer.fileformat.BibtexParser;
+import net.sf.jabref.logic.formatter.bibtexfields.UnicodeToLatexFormatter;
+import net.sf.jabref.logic.help.HelpFile;
+import net.sf.jabref.logic.importer.FetcherException;
+import net.sf.jabref.logic.importer.IdBasedFetcher;
+import net.sf.jabref.logic.net.URLDownload;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.FieldName;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+public class DiVA implements IdBasedFetcher {
+
+    private static final Log LOGGER = LogFactory.getLog(DiVA.class);
+
+    private static final String URL_PATTERN = "http://www.diva-portal.org/smash/getreferences?referenceFormat=BibTex&pids=%s";
+
+
+    @Override
+    public String getName() {
+        return "DiVA";
+    }
+
+    @Override
+    public HelpFile getHelpPage() {
+        return HelpFile.FETCHER_DIVA_TO_BIBTEX;
+    }
+
```

Comment:
I have no idea what the requirements are. Most I've seen are on the format `diva2:XXXXX` though, where X is a digit.